src/EventStore.Core/Services/Transport/Http/Authentication/ClientCertificateAuthenticationProvider.cs

```diff
@@ -20,14 +21,22 @@ public class ClientCertificateAuthenticationProvider : IHttpAuthenticationProvid
 			if (clientCertificate is null) return false;
 
 			bool hasReservedNodeCN;
+			string clientCertificateCommonName;
 			try {
+				clientCertificateCommonName = clientCertificate.GetNameInfo(X509NameType.SimpleName, false);
 				hasReservedNodeCN = clientCertificate.GetNameInfo(X509NameType.SimpleName, false) == _certificateReservedNodeCommonName;
 			} catch (CryptographicException) {
 				return false;
 			} catch (NullReferenceException) {
 				return false;
 			}
 
+			if (!hasReservedNodeCN) {
+				Log.Error(
+					$"Connection from Node {context.Connection.RemoteIpAddress} has been denied because it's CN: {clientCertificateCommonName} does not match with expected CN: {_certificateReservedNodeCommonName}");
```

Comment:
```suggestion
				var ip = context.Connection.RemoteIpAddress?.ToString() ?? "<null>";
				Log.Error(
					"Connection from: {ip} has been denied because it's CN: {clientCertificateCN} does not match with the expected CN: {expectedCN}",
					ip, clientCertificateCommonName, _certificateReservedNodeCommonName);
```

Suggestions:
- remove `node` since we're not 100% sure it's a node connecting
- structured logging